src/components/store/store.tsx

```diff
@@ -0,0 +1,19 @@
+import { configureStore } from '@reduxjs/toolkit';
```

Comment:
Why the store is located inside the components? You should separate business logic from UI